From 0035ce09d999393a87d0737e177085b91b54320f Mon Sep 17 00:00:00 2001
From: tedyu <yuzhihong@gmail.com>
Date: Mon, 18 Jul 2016 10:16:48 -0700
Subject: [PATCH 256/362] HBASE-16237 Blocks for hbase:meta table are not
 cached in L1 cache (Lars George and Ted Yu)

Change-Id: I8711a6fb17b6e30a989b66028923cb77958cf1d9
Author: Ted Yu
Reason: Improvement
Ref: CDH-50436
---
 .../org/apache/hadoop/hbase/HTableDescriptor.java  |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hbase-client/src/main/java/org/apache/hadoop/hbase/HTableDescriptor.java b/hbase-client/src/main/java/org/apache/hadoop/hbase/HTableDescriptor.java
index 3a626f7..1ac18a3 100644
--- a/hbase-client/src/main/java/org/apache/hadoop/hbase/HTableDescriptor.java
+++ b/hbase-client/src/main/java/org/apache/hadoop/hbase/HTableDescriptor.java
@@ -1742,6 +1742,7 @@ public class HTableDescriptor implements WritableComparable<HTableDescriptor> {
           .setScope(HConstants.REPLICATION_SCOPE_LOCAL)
           // Disable blooms for meta.  Needs work.  Seems to mess w/ getClosestOrBefore.
           .setBloomFilterType(BloomType.NONE)
+          .setCacheDataInL1(true)
          });
     metaDescriptor.addCoprocessor(
       "org.apache.hadoop.hbase.coprocessor.MultiRowMutationEndpoint",
-- 
1.7.9.5

